### PR TITLE
Added Oshomo script

### PR DESCRIPTION
**Pull request by hoshomoh**

### CODE REVIEW

File: scripts/oshomo.oforomeh-v2.js
Comment (JibolaPeace) [inline scripts/oshomo.oforomeh-v2.js:1]: Why did you use a / in your script? i.e scripts/oshomo.oforomeh

**Review by LoluT — APPROVED** [review]
Nice one 💯

**Comment by Udemebilly** [comment]
Sorry i was in the wrong path!.... Nice Job!

**Comment by hoshomoh** [comment]
> Sorry i was in the wrong path!.... Nice Job!

PS: This is just a standalone comment. You didn't really approve it?

**Review by JibolaPeace — APPROVED** [review]

**Review by Udemebilly — APPROVED** [review]

**Review by Udemebilly — APPROVED** [review]
approval sent.

**Pull request merged**